app/models/unit.rb

```diff
@@ -223,23 +223,25 @@ def tutorial_with_abbr(abbr)
 
   #
   # Imports users into a project from CSV file.
-  # Format: unit_code, Student ID,First Name,Surname,email,tutorial
-  #
+  # Format: Subject/Unit Code, Student ID,First Name, Surname, email, tutorial
+  # Expected columns: unit_code, username, first_name, last_name, email, tutorial
   def import_users_from_csv(file)
     tutorial_cache = {}
     success = []
     errors = []
     ignored = []
 
-    CSV.foreach(file) do |row|
+    CSV.foreach(file, headers: true) do |row|
```

Comment:
Also add in the following to ensure that empty and strange UTF8 characters are handled ok...

```:header_converters => [lambda { |i| i.nil? ? '' : i },  :downcase, lambda { |hdr| hdr.strip unless hdr.nil?}],
        :converters => [lambda{ |body| body.encode!('UTF-8', 'binary', invalid: :replace, undef: :replace, replace: '') unless body.nil? }]```